code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -1,7 +1,7 @@
 #define CHALLENGE_TELECRYSTALS 280
-#define CHALLENGE_TIME_LIMIT 3000
+#define CHALLENGE_TIME_LIMIT 5 MINUTES
 #define CHALLENGE_MIN_PLAYERS 50
-#define CHALLENGE_SHUTTLE_DELAY 15000 // 25 minutes, so the ops have at least 5 minutes before the shuttle is callable.
+#define CHALLENGE_SHUTTLE_DELAY 35 MINUTES	//35 minutes, giving nukies at least 20 minutes to enact their assault.
```

Comment:
Not 15?

Reply:
10 minutes were added, as correctly described by the CL

20 minutes to assault, 15 mins for call
previously it was 20mins to assault, 5 mins for call

So the CL is correct, the comment was just changed.

Reply:
Both are correct, CL is the time until shuttle call, comment is about total time of attack